profiles/tests/test_models.py

```diff
@@ -173,27 +175,8 @@ def test_import_customer_data_with_missing_email():
         }
     ]
     assert Profile.objects.count() == 0
-    with pytest.raises(ProfileMustHaveOnePrimaryEmail) as e:
-        Profile.import_customer_data(data)
-    assert str(e.value) == "Profile must have exactly one primary email, index: 0"
-    assert Profile.objects.count() == 0
-
-
-def test_validation_should_pass_with_one_primary_email():
-    profile = ProfileWithPrimaryEmailFactory()
-    validate_primary_email(profile)
-
-
-def test_validation_should_fail_with_no_primary_email(profile):
-    with pytest.raises(ProfileMustHaveOnePrimaryEmail):
-        validate_primary_email(profile)
-
-
-def test_validation_should_fail_with_multiple_primary_emails(profile):
-    EmailFactory(profile=profile, primary=True)
-    EmailFactory(profile=profile, primary=True)
-    with pytest.raises(ProfileMustHaveOnePrimaryEmail):
-        validate_primary_email(profile)
```

Comment:
I suppose we don't want to allow more than one primary email still.

Reply:
It was ensured in the admin, GQL, and the importer. But I now added the check to the Profile model too.

Reply:
Although I commented on some place in the code, I intention was that it should be ensured that by no means it's possible to create multiple primary emails for a profile. Perhaps my main concern was the GraphQL API, although the test code I inserted the comment on isn't directly about the GraphQL API. This was badly communicated by me.

But what led to the comment was that it turns out that I wasn't quite aware of how the GraphQL works when multiple emails are set as primary. For example the GraphQL query:

```graphql
mutation {
  updateMyProfile(input: {
    profile: {
      addEmails: [
        {
          email: "new_email@example.com",
          emailType: OTHER
          primary: true
        }
      ]
      updateEmails: [
        {
          id: "AAAA",
          primary: true
        },
        {
          id: "BBBB",
          primary: true
        },
      ]
    }
  }) {
    profile {
      ...
    }
  }
}
```

I thought that this would result into some error, telling the user not try to set multiple emails as primary. But it seems that instead, the query (was and still) is accepted, and whatever email happens to be the last one processed, wins and will be the primary email. A bit weird design, but it is what it is.

So, based on that, my original concern wasn't that valid after all.

Reply:
In other words, as the `PROFILE_MUST_HAVE_ONE_PRIMARY_EMAIL` error code was removed in this PR, I was expecting that there should now be some `PROFILE_CAN_NOT_HAVE_MORE_THAN_ONE_PRIMARY_EMAILS` error code. But now I have learned, that there doesn't have to be.